projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -81,7 +81,7 @@ module.exports = async function (...args) {
 	}
 
 	const runLegacyBuild =
-		!FEDERATION_CONFIG || FEDERATION_CONFIG.runLegacyBuild !== false;
+		!federation || ['compatible', 'disabled'].includes(federation.mode);
```

Comment:
Good naming :sweat_smile: . I wrote `incompatible` because the other option is `compatible`, but being the default we may as well name it `default` :+1: .

Reply:
And `compatible` is because it is compatible with the previous architecture (i.e.: bundler + AMD).

Reply:
> but being the default we may as well name it `default` 👍 .

Sounds a bit easier to sell than `incompatible`. 😁